python/dgl/batch.py

```diff
@@ -4,12 +4,12 @@
 from . import backend as F
 from .base import ALL, is_all, DGLError, dgl_warning, NID, EID
 from .heterograph_index import disjoint_union, slice_gidx
-from .heterograph import DGLHeteroGraph
+from .heterograph import DGLGraph
 from . import convert
 from . import utils
 
 
-__all__ = ['batch', 'unbatch', 'slice_batch', 'batch_hetero', 'unbatch_hetero']
```

Comment:
It seems that you also deprecate `batch_hetero` and `unbatch_hetero` in this PR. Make sure you state that in the PR description and link appropriate tracker items.

Reply:
Revert it in this PR as it beyond scope.